tools/autograd/derivatives.yaml

```diff
@@ -658,6 +658,9 @@
 - name: embedding(Tensor weight, Tensor indices, int64_t padding_idx, bool scale_grad_by_freq, bool sparse)
   weight: embedding_backward(grad, indices, weight.size(0), padding_idx, scale_grad_by_freq, sparse)
 
+- name: embedding_bag(Tensor weight, Tensor indices, Tensor offsets, Tensor offset2bag, Tensor bag_size, bool scale_grad_by_freq, int64_t mode, bool sparse)
+  weight: embedding_bag_backward(grad, indices, offsets, offset2bag, bag_size, weight.size(0), scale_grad_by_freq, mode, sparse)
```

Comment:
(output, offset2bag, bag_size)

result1 = offset2bag

result2 = bag_size

check the comment at the top of file